diesel/src/mysql/types/date_and_time.rs

```diff
@@ -164,7 +164,7 @@ mod tests {
     use crate::sql_types::{Date, Datetime, Time, Timestamp};
 
     fn connection() -> MysqlConnection {
-        dotenv().ok();
+        let _ = dotenv();
```

Comment:
I would wait with applying this change till the documentation of `dotenv()` itself recommends this: https://github.com/dotenv-rs/dotenv/pull/57

Reply:
@weiznich I've addressed your first comment. Now, I'll remove all the `dotenv()` changes, I will make another PR once that lands.